docs/ops/generation/RandomUniform_8.md

```diff
@@ -175,18 +177,20 @@ output  = [[65 70 56]
     * **Type**: string
     * **Required**: *Yes*
 
-* *seed*
+* *global_seed*
 
     * **Description**: global seed value.
     * **Range of values**: positive integers
     * **Type**: `int`
+    * **Default value**: 0
```

Comment:
Now it is misaligned with nGraph implementation where these values are mandatory based on the constructor

Reply:
No, shell implementation was corrected as Michal Karzynski asked to add default values. 
RandomUniform constructor: https://github.com/openvinotoolkit/openvino/blob/2c4009e3d8e4c382341c1bcbef9871a8d0390c8d/ngraph/core/include/openvino/op/random_uniform.hpp#L28

In this PR I align it with specification.